models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,82 @@
+# colorization-v2-norebal
```

Comment:
Add link to license in last section `Legal Information`

Reply:
@eizamaliev done